FIRO_TSEnsembles/src/main/java/hec/metrics/MetricsConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package hec.metrics;
+
+import hec.stats.Configuration;
+
+import java.time.Duration;
+import java.time.ZonedDateTime;
+
+public class MetricsConfiguration  implements Configuration {
+    private final Duration _interval;
+    private final ZonedDateTime _issueDate;
+    private final ZonedDateTime _startDateTime;
+    private final String _units;
+
+    private MetricsConfiguration(ZonedDateTime issueDate, ZonedDateTime startDate, Duration interval, String units)
```

Comment:
Is this a configuration or a Definition?  if yes then could this be MetricsDefinition

Reply:
This does need work, i think that configuration needs to be better defined, it is used in ensembles and in metrics, and may not be appropriate for both.